api/turing/cluster/servicebuilder/router.go

```diff
@@ -503,10 +504,22 @@ func buildFiberConfigMap(
 	// then define root-level fiber component as a lazy router with
 	// a traffic-splitting strategy based on these rules
 	if ver.TrafficRules != nil && len(ver.TrafficRules) > 0 {
+		// TrafficRule struct used requires the name and conditions field to be specified. But
+		// Default Traffic Rule has no name and a hardcoded name can be used instead since
+		// the name field is not used for traffic splitting strategy. Likewise, an empty slice
+		// of conditions can be used for the same reason.
+		rules := append(
+			ver.TrafficRules,
+			&models.TrafficRule{
+				Name:       "default-traffic-rule",
```

Comment:
It makes sense to have a name as there is no difference between the default and custom rules from the router's POV, for the sake of metrics and logs. This being the case, I think the API & UI should validate that the same name (although extremely unlikely) is not somehow set by the user.

Reply:
Added validation to both API and UI, screenshot in description!